client_tools.py

```diff
@@ -46,13 +45,17 @@ def parse_command_line():
 
     search_parser.add_argument('term')
 
```

Comment:
new format for date arg for picturae_import to match picturaes image folder date format